### PR TITLE
Update model diagram

### PR DESCRIPTION
**Pull request by benclmnt**
Changes:

- Added plantuml github actions

### CODE REVIEW

**Comment by benclmnt** [comment]
Please help me check if this class diagram is correct 😃 

image

**Review by dhafinrazaq — COMMENTED** [review]
Nevermind this comment

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -391,7 +478,31 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+    
+**Extensions**
+
+* 1a. The session overlaps with an existing Session.
+    
+    * 1a1. FitEgo shows an error message.
+
+      Use case ends.
+      
+* 1b. The Session is missing some required details.
```
Comment (dhafinrazaq) [inline docs/DeveloperGuide.md:577]: Would this mean we need to have this extension in every command? (extension if we have a missing argument)

Comment (benclmnt) [reply docs/DeveloperGuide.md:577]: should we?

**Review by tanweijie123 — APPROVED** [review]
👍

**Pull request merged**